### PR TITLE
Added a few tests

### PR DESCRIPTION
**Pull request by sircelsius**
testing

### CODE REVIEW

**Comment by sircelsius** [comment]
Hey @gauntface this is an early attempt at solving #778

Could you have a quick look and tell me if I'm on the right path here?

I added `get` and `post` to the `workbox-sw` router and not the `workbox-routing` base class, is that what you meant?

Also, `gulp lint test` fails on my system (node 8.4.0, npm 5.3.0) with the following error:

image


I'm no expert but it looks like some issue with the filesystem, are you familiar with it?

**Comment by gauntface** [comment]
@sircelsius sorry but we haven't reached any consensus on that issue, more just discussing the idea sorry :(

**Comment by sircelsius** [comment]
No probs at all, I'm more experimenting to try to understand how you guys would implement it than contributing (hence the PR against my fork rather than the base project) 😺

File: packages/workbox-sw/test/sw/router.js [hunk 1]
```diff
@@ -157,4 +157,19 @@ describe(`Test workboxSW.router`, function() {
     const route = workboxSW.router.registerRoute(() => {}, () => {});
     expect(route).to.exist;
   });
+
+  it(`should support calling get()`, function() {
+    const workboxSW = new WorkboxSW();
+    return new Promise((resolve) => {
+      const route = workboxSW.get('/', (args) => resolve());
```
Comment (gauntface) [inline packages/workbox-sw/test/sw/router.js:164]: Calling resolve here will return the test as a success before do the expect checks. You'll probably want to add the route checks to `new Promise().then(() => { /*checks here */});`

**Comment by gauntface** [comment]
Got ya :)

The issue looks like one of the node modules is failing to install. Maybe try removing and re-installing node_modules.

File: packages/workbox-sw/test/sw/router.js (hunk 1 above)
Comment (sircelsius) [reply packages/workbox-sw/test/sw/router.js:164]: Even if I'm only doing static checks on the `route` object itself (see below)?

I adapted [this previous test](https://github.com/GoogleChrome/workbox/blob/master/packages/workbox-sw/test/sw/router.js#L80) from the same file.

Comment (gauntface) [reply packages/workbox-sw/test/sw/router.js:164]: You are right, although if that's all you are doing, then you don't need the outer promise at all or the FetchEvent etc.

**Pull request closed without merging**